apitest/src/main/java/bisq/apitest/linux/BitcoinDaemon.java

```diff
@@ -55,6 +55,7 @@ public void start() throws InterruptedException, IOException {
                 + " -txindex=1"
                 + " -peerbloomfilters=1"
                 + " -debug=net"
+                + " -fallbackfee=0.0002"
```

Comment:
This change seems to change the behaviour of bitcoind in mainnet mode. Before 0.20.0 bitcoind had different default `fallbackfee`s for mainnet and test modes (0 and 0.0002 respectively), which was then fixed by setting the defaults to 0 in all cases.

By setting `fallbackfee=0.0002` we're setting it on all net modes. I can't tell if that's a problem.

Reply:
Oh wait. This bitcoind will never run in mainnet, will it? Then the concern is that the apitest testing harness will have a different bitcoind environment than a production daemon would.

Again, I don't fully understand the significance of the `fallbackfee` option. It's used when the bitcoind can't estimate a "floating fee", right? And it only applies when sending funds from a wallet managed by the bitcoind instance, right? If both answers are yes, then I don't see this causing differences between prod and dev environments, but I'd like someone else to sanity check this too.

Reply:
This new fallbackfee setting would change the behaviour of bitcoind in mainnet mode -- from the default 0.00 to 0.0002.  But `apitest` cannot run on mainnet unless the code is hacked, and `apitest` should never run on mainnet -- it should never spend mainnet BTC, do a trade on mainnet, etc.

>> Then the concern is that the apitest testing harness will have a different bitcoind environment than a production daemon would.

Yes, it does mean that the `apitest` harness will have a different `bitcoind` environment than a production daemon would, but that is what we want.  The bitcoin-core change ([PR 16524](https://github.com/bitcoin/bitcoin/pull/16524)) was to make the fallbackfee the same on all chains -- to fix bitcoin-core's inconsistency.  It was 0 (disabled) on mainnet, but 0.0002 for testnet/regtest chains.  Now (v0.20+), it is the same for all chains: 0 (disabled) by default for all chains.

We want bitcoin-core v0.20 to behave like it did before the bitcoin-core PR 16524 change.

In the context of `apitest`, the fallbackfee affects the behaviour of the bitcoin-cli `sendtoaddress` command, where we use the `subtractfeefromamount=false` parameter.  (See `FundWalletScenarioTest#testFundWallet()`)  This `bitcoin-cli` command errors if the fallbackfee is disabled on bitcoin-core v0.20.  Enabling it on `bitcoind` ensures this command behaves the same way it does on v0.19.